lotto/offchain/src/Scenarii.hs

```diff
@@ -238,7 +238,7 @@ doubleSatisfaction setup salt secret = do
                 -- potentially be a lot; basically one of the pots minus 10 ADA.
                 Cooked.paysPK (Cooked.walletPKHash alice) restValue
               ],
-            Cooked.txSkelValidityRange = validityRange,
+            --Cooked.txSkelValidityRange = validityRange,
```

Comment:
Hum. Any particular reason? Also, the binding of `validityRange` line 220 is now useless, I suppose.